tests/unit/utils/parse-response-headers-test.js

```diff
@@ -92,7 +94,7 @@ test('ignores headers that do not contain a colon', function(assert) {
 
   let headers = parseResponseHeaders(headersString);
 
-  assert.deepEqual(headers['Content-Encoding'], 'gzip', 'parses basic header pair');
+  assert.deepEqual(headers['content-encoding'], 'gzip', 'parses basic header pair');
   assert.deepEqual(headers['apple'], 'pie', 'parses basic header pair');
   assert.equal(Object.keys(headers).length, 2, 'only has the one valid header');
```

Comment:
This assert's description is wrong (we had two and now we have three valid headers lol). The valid headers will be `content-encoding` `Content-Encoding` and `apple`